docs/source/user_guide/gui.md

```diff
@@ -0,0 +1,194 @@
+(target-gui)=
+# Graphical User Interface
+
+The `movement` graphical user interface (GUI), powered by our custom plugin for
+[napari](napari:), makes it easy to view and explore `movement`
+motion tracks. Currently, you can use it to
+visualise 2D [poses datasets](target-poses-and-bboxes-dataset)
+as points overlaid on video frames.
+
+:::{warning}
```

Comment:
Wdyt about adding a [link to issues tagged with the GUI label](https://github.com/neuroinformatics-unit/movement/issues?q=is%3Aissue%20state%3Aopen%20label%3AGUI) here so that we can merge this docs PR and investigate the issues in another?

Reply:
Yeah excellent idea, thanks for suggesting @lochhh and for planning @niksirbi 🚀 

I can continue to inspect this further and report any findings in issue #446

Reply:
Sounds good to me, although I'd prefer merging napari-dev to main sooner rather than waiting for all bugs to be fixed - we'll always discover more 🙈

Reply:
I wrote my last message before reading your replies, sound like we all agree, I will start the untangling now.

Reply:
Bold it is!